### PR TITLE
chore: add demos

### PR DESCRIPTION
**Pull request by purpose233**
[[中文版模板 / Chinese template](https://github.com/VisActor/vstory/blob/main/.github/PULL_REQUEST_TEMPLATE/pr_cn.md?plain=1)]

### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Refactoring
- [ ] Update dependency
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Site / documentation update
- [x] Demo update
- [ ] Workflow
- [ ] Release
- [ ] Other (about what?)

### 💡 Background and solution

add demos

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |           |
| 🇨🇳 Chinese |           |

### ☑️ Self-Check before Merge

⚠️ Please check all items below before requesting a reviewing. ⚠️

- [ ] Doc is updated/provided or not needed
- [ ] Demo is updated/provided or not needed
- [ ] TypeScript definition is updated/provided or not needed
- [ ] Changelog is provided or not needed

---

### 🚀 Summary

copilot:summary

### 🔍 Walkthrough

copilot:walkthrough

### CODE REVIEW

File: docs/assets/examples/en/infographic/bar-chart-horizontal.md [hunk 1]
```diff
@@ -0,0 +1,232 @@
...
+      id: 'defaultAct',
+      scenes: [
+        {
+          id: 'defaultScene',
+          actions: [
+            {
+              characterId: ['bgImage', 'bg0', 'bg1', 'title', 'chart', 'displayImage'],
+              characterActions: [
+                {
+                  action: 'appear'
```
Comment (xiaoluoHe) [inline docs/assets/examples/en/infographic/bar-chart-horizontal.md:28]: 加一下动画

File: docs/assets/examples/en/infographic/pie-chart-camera.md [hunk 2]
```diff
@@ -0,0 +1,231 @@
+---
+category: examples
+group: infographic
+title: Pie Chart Camera
+keywords: templates, visualization, bar, left-right, comparison, horizontal
```
Comment (xiaoluoHe) [inline docs/assets/examples/en/infographic/pie-chart-camera.md:5]: 关键词改一下吧，bar -> pie， left-right 不用了， comparison -> percentage

File: docs/assets/examples/en/infographic/pie-chart-camera.md [hunk 3]
```diff
@@ -0,0 +1,231 @@
...
+      scenes: [
+        {
+import { CustomInfographic } from './demos/infographic/bar';
+          id: 'defaultScene',
+          actions: [
+            {
+              characterId: ['bgImage', 'cameraIcon', 'bg0', 'bg1', 'title', 'chart', 'displayImage'],
+              characterActions: [
+                {
+                  action: 'appear'
```
Comment (xiaoluoHe) [inline docs/assets/examples/en/infographic/pie-chart-camera.md:29]: 同上，加一下动画

File: docs/assets/examples/en/infographic/pie-chart-camera.md (hunk 2 above)
Comment (purpose233) [reply docs/assets/examples/en/infographic/pie-chart-camera.md]: Fixed.

File: docs/assets/examples/en/infographic/pie-chart-camera.md (hunk 3 above)
Comment (purpose233) [reply docs/assets/examples/en/infographic/pie-chart-camera.md]: Fixed.

File: docs/assets/examples/en/infographic/bar-chart-horizontal.md (hunk 1 above)
Comment (purpose233) [reply docs/assets/examples/en/infographic/bar-chart-horizontal.md]: Fixed.

**Review by xiaoluoHe — APPROVED** [review]

**Pull request merged**